receiver/README.md

```diff
@@ -95,22 +95,32 @@ receivers:
 
 ## <a name="jaeger"></a>Jaeger Receiver
 **Only traces are supported.**
-// TODO(ccaraman) Update Jaeger receiver documentation.
 
-This receiver receives spans from Jaeger collector HTTP and Thrift uploads and translates them into the internal span types that are then sent to the collector/exporters.
-Only traces are supported. This receiver does not support metrics.
+This receiver receives traces from [Jaeger](https://www.jaegertracing.io)
```

Comment:
I think it will be clear to say that it receives trace data in the Jaeger format, no need to talk about the source of the data.

Reply:
The line is a bit redundant, but I like that it clearly links to the jaeger documentation. 

One benefit of having this line for each receiver is users can quickly look at the various instrumetations supported by otel. 

I did the same thing with OC receiver.

Reply:
thinking about it use your wording and keep the link to jaeger.